src/main/java/com/google/opensesame/github/GitHubGetter.java

```diff
@@ -8,7 +8,11 @@
   private static GitHub gitHub = null;
 
   /**
-   * Gets the GitHub API interface or builds one if there is not one available.
+   * Gets the GitHub API interface or builds one if there is not one available. If credentials are
+   * available in the system environment variables, the builder will connect to the GitHub API using
+   * those. Otherwise, it will connect anonymously.
+   *
+   * <p>In AppEngine, the credentials can be set in appengine-web.xml.
```

Comment:
```suggestion
   * In AppEngine, the credentials can be set in appengine-web.xml.
```

Reply:
Yeah I agree with this change. For some reason this was being enforced by the Java linter.. Although it seems to have stopped now? Hm.

Reply:
Yeah the Java linter added it back. Any thoughts on why that might happen? Seems odd.